tasks/message-queue-consumers.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: "Get the list of magento message queue message consumers"
+  become: "yes"
+  become_user: "{{ magento_user }}"
+  command: "/usr/bin/php {{ magento_app_root }}/bin/magento queue:consumers:list"
+  register: magento_queue_consumers_list_output
+
+- set_fact:
+    code: "magento_mq_{{ item | regex_replace('[^A-Za-z]', '_') | lower }}"
+    name: "{{ item }}"
+  with_items: "{{ magento_queue_consumers_list_output.stdout_lines }}"
+  register: magento_queue_consumers_list_wrapped
+
+- set_fact:
+    magento_queue_consumers_list: "{{ magento_queue_consumers_list_wrapped|json_query('results[*].ansible_facts') }}"
+
+- debug:
```

Comment:
Do we really want debug statements here? And if so, could we add "name" description so that it's easier to find in crazy long output in pipeline?

Reply:
true, well `debug` is also an output tool, I'm using it here to list the cron tasks that are going to be installed, I will add a speaking name here

Reply:
updated